src/libraries/System.Private.Xml.Linq/tests/XDocument.Common/BridgeHelpers.cs

```diff
@@ -539,42 +539,51 @@ public static void DeleteTestFile(string strFileName)
 
         public static void CreateByteTestFile(string strFileName)
         {
-            FilePathUtil.addStream(strFileName, new MemoryStream());
-            TextWriter tw = new StreamWriter(FilePathUtil.getStream(strFileName));
+            var ms = new MemoryStream();
```

Comment:
If I'm understanding the nature of these fixes correctly, these tests used cache values globally before they'd been fully populated?

Reply:
before @danmoseley's change the tests used cache in very non-intuitive way which made it hard to reason if they used cached or original version. @danmoseley has made it so that we always create a copy of what was added and this follows up on that change. Here previously we relied on the fact that it could modify original which is bad design to start with